docs/faq.md

```diff
@@ -99,4 +99,76 @@ For reference, check out [windicss](https://github.com/windicss/vite-plugin-wind
 
 ### What is going on with Vite and `Pre-bundling dependencies:`?
 
-Pre-bundling dependencies is an [optimization in Vite](https://vitejs.dev/guide/dep-pre-bundling.html). It is required for CJS dependencies, as Vite's development server only works with ES modules on the client side. Importantly for Svelte libraries and ESM modules, prebundling combines component libraries into a single file to speed up the initial page load. Try setting the [`prebundleSvelteLibraries`](./config.md#prebundleSvelteLibraries) option to `true` to speed things up. This will likely be enabled by default in future version of the plugin.
+Prebundling dependencies is an [optimization in Vite](https://vitejs.dev/guide/dep-pre-bundling.html).
+
+> We only use prebundling during **development**, the following does not apply to or describe the built application
+
+It is required for CJS dependencies, as Vite's development server only works with ES modules on the client side.
+Importantly for Svelte libraries and ES modules, it also reduces the number of http requests when you load a page from the dev server and caches files so subsequent starts are even faster.
+
+The way prebundling Svelte libraries affects your dev-server load times depends on the import style you use, index or deep:
+
+#### index imports
+
+offers better DX but can cause noticable delays on your machine, esp. for libraries with many files.
+
+```diff
+import {SomeComponent} from 'some-library'
++ only one request per library
++ intellisense for the whole library after first import
+- compiles the whole library even if you only use a few components
+- slower build and dev-server ssr
+```
+
+#### deep imports
+
+offers snappier dev and faster builds for libraries with many files at the expense of some DX
+
+```diff
+import SomeComponent from 'some-library/src/SomeComponent.svelte'
++ compiles only the components you import
++ faster build and dev-server ssr
+- one request per import can slow down initial load if you use a lot of components
+- intellisense only for imported components
+```
+
+#### a word on import (re-)writing with plugins or preprocessors (auto-import, optimizeImports etc)
```

Comment:
```suggestion
#### Rewriting imports with plugins or preprocessors
```

Maybe this could be shorten a bit 🤔  Would be nice to have the example libraries linked in the paragraph below too.